src/nitrokey/nk3/updates.py

```diff
@@ -119,10 +119,6 @@ def confirm_extra_information(self, extra_info: List[str]) -> None:
     def confirm_update_same_version(self, version: Version) -> None:
         pass
 
-    @abstractmethod
-    def request_repeated_update(self) -> Optional[Exception]:
-        pass
```

Comment:
Can we just remove `@abstractmethod` and trigger a [`DeprecationWarning`](https://docs.python.org/3.13/library/exceptions.html#DeprecationWarning) in its body so that this is a non-breaking change?